physionet-django/events/models.py

```diff
@@ -153,3 +155,33 @@ def withdraw(self, comment_to_applicant=None):
             comment_to_applicant (str): The comment to add to the participant about the status.
         """
         self._apply_decision(self.EventApplicationStatus.WITHDRAWN, comment_to_applicant)
+
+
+class EventAgreement(models.Model):
```

Comment:
Now that we have a couple of different types of agreement, I'm wondering whether we should have a generic "Agreement" model (and then either inherit from that model, or have a new agreeement_type attribute. Any thoughts on this? Also q for @bemoody

Reply:
@tompollard Thats a good point. I like the 2nd idea `new agreeement_type attribute` with text enums.

We could add the attribute to the DUA model, make the default='Project' or something like that, and slightly change `CreateProjectForm` and related views.
and can use it directly in Events.

we will probably be able to do it after the Datathon(but weirdly it will be a bad decision to merge the current PR if we decide to go with the `new_agreement_type` attribute as we would be adding a table to database, just to remove it next week)

Reply:
Both Benjamin and Alistair pointed out the benefit of not introducing dependencies between apps, so let's ignore this idea for now!